.script/tests/KqlvalidationsTests/CustomTables/ForescoutHostProperties_CL.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
We do not have data connectors. Instead we use Sentinel REST api to upload data and download incidents.

Reply:
How do you run these calls to Sentinel REST API? Is it using an Azure Function or logic app or something else?
This is useful for us to know since customers will not know how to ingest data if we do not have any data connector.

Reply:
We have an application (which is available only to customers) that makes use of Sentinel REST API to upload and download.

Reply:
@fs-connect Thanks for the clarification. How can other users, apart from your customers ingestion this data into Sentinel? If we have clearly defined steps, can you document it in the data connector format applicable to this product. You can find all the templates here - https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors/Templates

Reply:
This is a "customer only" product. So, only customers can ingest data into the custom logs.

Reply:
Got it.